examples/multiclass/plot_multiclass_overview.py

```diff
@@ -0,0 +1,196 @@
+"""
+===============================================
+Overview of multiclass training meta-estimators
+===============================================
+
+In this example, we discuss the problem of classification when the target
+variable is composed of more than two classes. This is called multiclass
+classification.
+
+In scikit-learn, all estimators support multiclass classification out of the
+box: the most sensible strategy was implemented for the end-user. The
+:mod:`sklearn.multiclass` module implements various strategies that
+one can use for experimenting. :mod:`sklearn.multiclass` includes OvO/OvR
```

Comment:
I always considered the `multiclass` module useful for third party estimators that only support binary classification. If you agree, can you add the statement here?